src/helpers/copyToClipboard.ts

```diff
@@ -0,0 +1,5 @@
+export const copyToClipboard = (text:string) => {
+  navigator.clipboard.writeText(text)
+    .then(() => console.log('Copiado al portapapeles'))
+    .catch((error) => console.error('Error al copiar al portapapeles: ', error));
```

Comment:
Tal vez tendrías que retornar dicha premisa para que en el `handleCopyText` puedas envolver esta función `copyToClipboard` en un try-catch. Para que igual puedas cachar las excepciones y con base a eso mostrar algún error

```suggestion
     return navigator.clipboard.writeText(text)
```